ddev/src/ddev/cli/validate/licenses.py

```diff
@@ -0,0 +1,49 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import click
+from packaging.requirements import Requirement
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+
+@click.command(short_help='Validate third-party license list')
+@click.option('--sync', '-s', is_flag=True, help='Generate the `LICENSE-3rdparty.csv` file')
+@click.pass_context
+def licenses(ctx: click.Context, sync):
+    app: Application = ctx.obj
```

Comment:
```suggestion
@click.pass_obj
def licenses(app: Application, sync: bool):
```

Reply:
Ah TIL

Reply:
Whoops actually I still use `ctx.invoked()` at the end of this function, so for now I cannot directly use `app`.